cucumber.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'cucumber-html-formatter', '>= 19.1', '< 21.0'
   s.add_dependency 'cucumber-messages', '>= 18', '< 22'
   s.add_dependency 'diff-lcs', '~> 1.5', '>= 1.5.0'
-  s.add_dependency 'mime-types', '~> 3.4', '>= 3.4.1'
+  s.add_dependency 'mini_mime', '~> 1'
```

Comment:
can we lock this to 1.0 then it's rtm. Sorry about the confusion with the caching thing

Reply:
Awesome! Thanks.

I've fixed this with a rebase: [`e021f50`](https://github.com/cucumber/cucumber-ruby/compare/b635fa51fb635d122b70a680acbd355b035bac4d..e021f50052a1fc1721585b74a8991df507f8ac7e)